### ISSUE TITLE
[UI - Dialogue] Right-click to Rewind Text/Dialogue Windows

### ISSUE THREAD

**Issue by robertkety** [LateStartStudio/Hero6#221]
There was nothing I hated more as a kid than accidentally clicking through dialogue.  I always thought that the right mouse button should act as an undo or back button.  (That wouldn't have any affect on spoken dialogue, just written).

**Comment by brianjlacy** [comment]
I agree, this is a common frustration for me as well. But perhaps a simpler alternative would suffice -- a "journal" or "quest log" that tracks all displayed text, which can be displayed by selecting an option in the HUD..?
